src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/overview/overview.component.html

```diff
@@ -1,5 +1,23 @@
 <cd-view-cache [status]="status"></cd-view-cache>
 
+<div class="row">
+  <div class="col-md-10">
+    <table class="table-striped">
+      <tr>
+        <td i18n><strong>Site Name:&nbsp;</strong></td>
+        <td class="w-75">{{ siteName }}</td>
+      </tr>
+    </table>
+  </div>
+  <div class="col-md-2 text-right">
+    <cd-table-actions class="table-actions"
+                      [permission]="permission"
+                      [selection]="selection"
+                      [tableActions]="tableActions">
+    </cd-table-actions>
+  </div>
+</div>
```

Comment:
Current code is breaking the site name in 2 rows and will break the button if the screen is smaller.
I think the following works well in all situation:
```
<div class="row">
  <div class="col-md-12">
    <span><strong i18n>Site Name:</strong> {{ siteName }}</span>
    <cd-table-actions class="table-actions float-right"
                      [permission]="permission"
                      [selection]="selection"
                      [tableActions]="tableActions">
    </cd-table-actions>
  </div>
</div>
```

old @ 1024px:
![image](https://user-images.githubusercontent.com/399326/68688397-2ee44d00-055e-11ea-920a-91607aa85f78.png)
new:
![image](https://user-images.githubusercontent.com/399326/68688446-44597700-055e-11ea-9093-1bf7368f35d1.png)